src/acceptance-test/java/net/consensys/orion/acceptance/send/SingleNodeSendTest.java

```diff
@@ -131,7 +131,7 @@ void emptyPrivacyGroup() {
 
     final String response = sendPrivacyGroupTransactionExpectingError(orionNode, PK_1_B_64, "");
 
-    assertError(OrionErrorCode.INVALID_PAYLOAD, response);
+    assertError(OrionErrorCode.ENCLAVE_PRIVACY_GROUP_MISSING, response);
```

Comment:
Asserting that a payload is invalid for an empty privacyGroupId doesn't really make sense. When we try to find the privacyGroupId with id = "", it should not find any and return that the group is missing.

It has nothing to do with the payload.

Reply:
INVALID_PAYLOAD could be ok if we have a definition of "privacyGroupId must be 32 bytes" ...